object/type.go

```diff
@@ -16,11 +16,13 @@ const (
 )
 
 // ToV2 converts [Type] to v2 [object.Type].
+// Deprecated: do not use it.
```

Comment:
Should I use something else instead?

Reply:
simple cast. But you shouldnt need this action at all

Reply:
People can appreciate some additional comments on this, otherwise it's like "hey, I'm using it and now I can't".

Reply:
pleased those ppl